### DISCUSSION TITLE
Please, help to find documentation to understand and learn deeply  understrap file system especially the  /inc folder.

### DISCUSSION THREAD

**Discussion by ddcddc** [holger1411/understrap#2157] [General]
I work with understrap  since 5 or 6 years but I still spending hundred and hundred hours trying and discovering  things that surely are basics skiils.

I love this theme. Id'like to get wide perspective about every files, folders, **guidelines**, bests practices for dev.

My workflow is creating a child-theme and compiling scss with npm-watch and diving in the sea of variables.scss, wp native variables, etc etc.

**Comment by bacoords** [comment]
I would also recommend reading the docs which cover a few of those things: https://docs.understrap.com/#/

**Reply by ddcddc** [reply]
I notice that I have to make a lot of scss changes and it takes about 10 - 15s to compile.
I'm looking for make the compilation processes more efficient, eliminating some during development like minify.

Anyway, I'd like learn how you should do development with npm in understrap is a excuse for learn npm.
If anyone could give me some tips that would be great.

Openai is good, but doesn't give me good advices :-)
